### ISSUE TITLE
advice on a regression problem similar to the one in chapter 8 section 3

### ISSUE THREAD

**Issue by pasq-cat** [tensorchiefs/dl_book#12]
Hi, first of all i would like to thank you for writing the book, i am reading it at the moment and i find it really interesting. I am a student learning how to use TensorFlow on a  scientific data augmentation problem. I modified the code presented in chapter 8.3 (regression with mcdropout) to create a Bayesian neural network that recreates the trend of a function based on a set of points (x,y) and provides an estimate of the error associated with each prediction.  The network works well and is quite accurate but the problem is that, just like in the example of the book, I assumed that the measures are independent of each other when in fact the data I have have a covariance matrix that has small but not null terms outside the diagonal.  I was wondering if there was a way in tensorflow_probability to modify the loss function to use a multivariate gaussian while leaving the output layer unchanged (I don't want to predict a whole new covariance matrix, just the mean and the associated error estimate).  Thanks for any advice.
